README.md

```diff
@@ -30,7 +30,7 @@ Let's jump in and get this up and running in Azure. When you are finished, you w
 |--|--|
```

Comment:
Maybe we should add some description on top of this table. 

"""
Before delving into the step-by-step execution of the application, you can simply click the Deploy to Azure button. This will instantly deploy the app to Azure Spring Apps.
"""